sycl/unittests/kernel-and-program/DeviceInfo.cpp

```diff
@@ -0,0 +1,89 @@
+//==-------------- DeviceInfo.cpp --- device info unit test ----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#define SYCL2020_DISABLE_DEPRECATION_WARNINGS
```

Comment:
Do we need this? Can we avoid using deprecated APIs in new tests?

Reply:
will remove it.